bot/exts/moderation/verification.py

```diff
@@ -156,6 +173,9 @@ class Verification(Cog):
     # ]
     task_cache = RedisCache()
 
+    # Cache who needs to receive an alternate verified DM.
```

Comment:
the imperative way you phrased this makes it sound like the next line caches something, when it actually just creates a cache.

Reply:
Addressed in ceffe46a0d5136308c8f0684c2c406dd34e758fb